metricbeat/module/traefik/health/health_test.go

```diff
@@ -72,3 +74,7 @@ func TestFetchEventContents(t *testing.T) {
 	assert.EqualValues(t, 17, statusCodes["200"])
 	assert.EqualValues(t, 1, statusCodes["404"])
 }
+
+func TestData(t *testing.T) {
+	mbtest.TestDataFiles(t, "traefik", "health", mbtest.TestDataConfig(t))
```

Comment:
Why `mbtest.TestDataConfig(t)` is not called directly from the `TestData` function which already has `t`?

Reply:
I did `TestDataFiles` to require a config, I did that so current `TestAll` can reuse the code, and there may be other things in the future like modules with multiple configs (not sure if makes sense).
`TestDataConfig(t)` is a helper to return the config of the current package.

But well though, actually we could have `TestDataFiles` to don't require a config and have another `TestDataFilesWithConfig` for the special cases.

I'll change this, thanks!